hazelcast/src/main/resources/hazelcast-config-5.3.json

```diff
@@ -4279,6 +4279,11 @@
           "type": "integer",
           "default": 0,
           "description": "Defines the timeout in milliseconds that is applied to queries without an explicit timeout."
+        },
+        "catalog-persistence-enabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "Sets whether SQL Catalog persistence is enabled for the node. With SQL Catalog persistence enabled you can restart the whole cluster without losing schema definition objects (such as MAPPINGs, TYPEs and VIEWs). The feature is implemented on top of the Hot Restart feature of Hazelcast which persists the data to disk. If enabled, you have to also configure Hot Restart. Feature is disabled by default. If you enable this option in open-source, the member will fail to start, you need Enterprise to run it and obtain a license from Hazelcast."
```

Comment:
this is a good description, I would add also Data Connections because that is important for HZ-2150